tests/v2/test_message_store_queue.nim

```diff
@@ -88,34 +88,6 @@ procSuite "Sorted store queue":
     check:
       store.len == capacity
 
-  test "Sender time can't be more than MaxTimeVariance in future":
```

Comment:
It is deleted because of the support for zero sender timestamp?

Reply:
No, this test case has been moved to the `test_waku_store.nim` module and extended with the following test cases:

```Nim
  asyncTest "it should store a message with no sender timestamp":
	# [...]

  asyncTest "it should not store a message with a sender time variance greater than max time variance (future)":
	# [...]

  asyncTest "it should not store a message with a sender time variance greater than max time variance (past)":
  # [...]
```

Since the logic, as this PRs title says, has been moved from the different store implementations to the waku store protocol.